travis/docker-thrift/thrift

```diff
@@ -2,7 +2,7 @@
 
 pwd=$(dirname ${PWD})
 
-THRIFT_VER=0.9.2
+THRIFT_VER=0.10.0
```

Comment:
Looks inconsistent with the version above, but there's no Docker image for 0.11.0 yet. The idea is then to keep the Java dependency at the latest and update the docker image once it gets published.

Reply:
Container should be soon there https://github.com/ahawkins/docker-thrift/issues/10

Reply:
Do you want to wait for that to be available, or should we merge this one and send an update once that becomes available? My personal preference is to get this merged now, as we don't seem to require 0.11 vs. 0.10.

Reply:
We don't have to wait. 
 
Should we update the thrift in other repos to be consistent? Although, It seems that different thrift versions are compatible on wire format.